main.go

```diff
@@ -110,11 +120,43 @@ func main() {
 	}
 
 	if !useTLS {
-		log.Info("serving gRPC-Gateway and OpenAPI Documentation on http://", gatewayAddr)
-		log.Fatalln(gwServer.ListenAndServe())
+		logger.Info("serving gRPC-Gateway and OpenAPI Documentation", zap.String("gatewayAddr", "http://"+gatewayAddr))
+		err = gwServer.ListenAndServe()
+		if err != nil {
+			logger.Fatal("failed to serve gRPC-Gateway", zap.Error(err), zap.Bool("tls", false))
+		}
```

Comment:
Same here, one line should be ok

Reply:
fixed fc4a972dc35840a06e1dd536ccfb1176527a749a